### ISSUE TITLE
Wrapping output in center tag causes toolbar misalignment

### ISSUE THREAD

**Issue by philippjfr** [bokeh/bokeh#4599]
As I've been updating our code to work with bokeh 0.12 I noticed that center aligning the notebook output using the html `center` tag causes misalignment of the toolbar. Here's a small code snippet that reproduces the issue:

``` python
from bokeh.io import notebook_div
from bokeh.models import HBox
from bokeh.plotting import figure
from IPython.display import display, HTML

output_notebook()

x = list(range(11))
y0 = x
y1 = [10 - i for i in x]
y2 = [abs(i - 5) for i in x]

# create a new plot
s1 = figure(width=300, plot_height=300, title=None)
s1.circle(x, y0, size=10, color="navy", alpha=0.5)

# create another one
s2 = figure(width=300, height=300, title=None)
s2.triangle(x, y1, size=10, color="firebrick", alpha=0.5)

# put all the plots in a VBox
p = HBox(children=[s1, s2])

# show the results
div = notebook_div(p)
display(HTML('<center>'+div+'</center>'))
```

![image](https://cloud.githubusercontent.com/assets/1550771/16204221/dab1f5f8-3715-11e6-88e1-bc597c5c8ab8.png)

Without the `<center>` tag it looks fine:

![image](https://cloud.githubusercontent.com/assets/1550771/16204267/0245338c-3716-11e6-87a7-6f1d0b43c351.png)

And I can get correct alignment by wrapping the output in a div like this `<div style="display:table; margin:0 auto;">`, so it isn't a massive priority.

This is using bokeh 0.12dev13 and Jupyter notebook 4.2.1.

**Comment by jbednar** [comment]
@bryevdv suggests tagging @birdsarah, so here goes.

**Comment by birdsarah** [comment]
Seems like something we should look into. Your workaround fix will likely break gridplot toolbars so I don't think it's a quick thing to stick into bokeh. But we definitely need some guidance around this.

What happens if you do `display(HTML('<div style="margin:auto">'+div+'</div>'))`

(as a side note `center` tag is deprecated in html5)

**Comment by birdsarah** [comment]
Also, can you try with the release candidate @jbednar, I doubt it'll make a different but there were some small tweaks related to toolbar in there.

**Comment by jbednar** [comment]
No change that I can see with rc1:

![image](https://cloud.githubusercontent.com/assets/1695496/16244946/2d1506e2-37c4-11e6-9e1d-05454af7409e.png)

**Comment by philippjfr** [comment]
``display(HTML('<div style="margin:auto">'+div+'</div>'))`` doesn't seem to center it, I haven't seen any issues with gridplot and ``<div style="display:table; margin:0 auto;">`` though.

**Comment by birdsarah** [comment]
@philippjfr have you tried all sizing modes and all toolbar positions?

**Comment by birdsarah** [comment]
And also not in notebook?

**Comment by bryevdv** [comment]
Just confirming that this is not fixed by any work up to 0.12.6dev7

Also, here is updated code:
```
from bokeh.io import notebook_div, output_notebook
from bokeh.layouts import row
from bokeh.plotting import figure
from IPython.display import display, HTML

output_notebook()

x = list(range(11))
y0 = x
y1 = [10 - i for i in x]
y2 = [abs(i - 5) for i in x]

# create a new plot
s1 = figure(width=300, plot_height=300, title=None)
s1.circle(x, y0, size=10, color="navy", alpha=0.5)

# create another one
s2 = figure(width=300, height=300, title=None)
s2.triangle(x, y1, size=10, color="firebrick", alpha=0.5)

# put all the plots in a VBox
p = row(children=[s1, s2])

# show the results
div = notebook_div(p)
display(HTML('<center>'+div+'</center>'))
```

**Comment by mattpap** [comment]
This is pretty trivial to hack around, though a proper fix would be to not rely on CSS (at least that much) for positioning within layout.
